test/versioned/express/segments.tap.js

```diff
@@ -872,10 +872,79 @@ test('when using a regular expression in path', function (t) {
   })
 })
 
-function setup(t) {
-  agent = helper.instrumentMockedAgent({
-    feature_flag: { express_segments: true }
+const codeLevelMetrics = [true, false]
```

Comment:
Do we want to add versioned tests for the other non Next.js frameworks since you found the right place to catch them all, or are you going to use the other JIRA tickets to handle that work?

Reply:
there are jira tickets to do that